source/renderer/app/containers/staking/dialogs/DelegationSetupWizardDialogContainer.js

```diff
@@ -202,6 +204,14 @@ export default class DelegationSetupWizardDialogContainer extends Component<
       this.handleIsWalletAcceptable(amount, reward)
     );
 
+    const hardwareWalletDeviceInfo = _getHardwareWalletDeviceInfoByWalletId(
+      selectedWallet.id
+    );
+
+    const isTrezor =
```

Comment:
This check is used in multiple places, it could be good to have a method in the store that receives a wallet id and checks it

Reply:
That is true, and I was also thinking about that. But, I decided to create a more robust function that will return more info's so we can reuse it in multiple places. But we can have both so I will add that